docs/explanation/release_process/validation_pipeline_execution.rst

```diff
@@ -101,3 +103,8 @@ There's multiple entities participating in the chain of validating a Checkbox sn
         deactivate TServer
         Jenkins-->>Pipeline: Update Pipeline with Job Outcome
         deactivate Pipeline
+
+.. add code format to link text
+.. |hwcert-jenkins-jobs| replace:: ``hwcert-jenkins-jobs``
+.. _hwcert-jenkins-jobs: https://github.com
```

Comment:
`canonical/hwcert-jenkins-jobs` is a private repository. If we decide to include it in the public doc, the link needs to be listed in `linkcheck_ignore` in `conf.py`

Reply:
Understood!
Indeed I'll change that to the full link and add it to the linkcheck_ignore list!